plugins/content/pagenavigation/pagenavigation.php

```diff
@@ -258,7 +258,7 @@ private static function getQueryDate($orderDate)
 
 			// Use created if publish_up is not set
 			case 'published' :
-				$queryDate = ' CASE WHEN a.publish_up = ' . $db->quote($db->getNullDate()) . ' THEN a.created ELSE a.publish_up END ';
+				$queryDate = ' CASE WHEN a.publish_up = IS NULL THEN a.created ELSE a.publish_up END ';
```

Comment:
and here

Reply:
ahhhhh seems i was blind when doing that.